apps/api/src/modules/job/job.service.spec.ts

```diff
@@ -355,149 +405,87 @@ describe('JobService', () => {
 
     describe('normal behavior', () => {
       it('should create the job successfully', async () => {
-        const now = new Date()
-
-        const MOCK_JOB = {
-          ...mock('job')
-            .omit(['castingId', 'createdAt', 'updatedAt', 'jobId', 'status'])
-            .get(),
-          castingId: MOCK_CASTING_ID,
-          shooting: mock('shooting').get(3),
-        }
-        MOCK_JOB.applicationDeadline = new Date()
-        MOCK_JOB.applicationDeadline.setFullYear(
-          MOCK_JOB.applicationDeadline.getFullYear() + 1,
-        )
-        // for each shooting set start date to be 2 years from now, set start time to be the same
-        MOCK_JOB.shooting.forEach((shooting) => {
-          shooting.startDate = new Date()
-          shooting.startDate.setFullYear(shooting.startDate.getFullYear() + 2)
-          shooting.startTime = now
-        })
-        // for each shooting set end date to be 3 years from now, set end time to be the same
-        MOCK_JOB.shooting.forEach((shooting) => {
-          shooting.endDate = new Date()
-          shooting.endDate.setFullYear(shooting.endDate.getFullYear() + 3)
-          shooting.endTime = now
-        })
-        // set actorCount, minAge and wage to be 1 if they are less than 1
-        if (MOCK_JOB.actorCount < 1) {
-          MOCK_JOB.actorCount = 1
-        }
-        if (MOCK_JOB.minAge < 1) {
-          MOCK_JOB.minAge = 1
-        }
-        if (MOCK_JOB.wage < 1) {
-          MOCK_JOB.wage = 1
-        }
+        const MOCK_JOB = createValidJob()
 
         const result = mock('job').get(1)[0]
-        console.log(result)
 
         jest.spyOn(repository, 'createJob').mockResolvedValue(result.jobId)
 
         const newId = await service.createJob(MOCK_JOB, MOCK_CASTING_ID)
 
         expect(repository.createJob).toBeCalledWith(MOCK_JOB, MOCK_CASTING_ID)
       })
+
+      it('should be bad request due to date conflict', async () => {
+        const MOCK_JOB = createValidJob()
+
+        const result = mock('job').get(1)[0]
```

Comment:
```suggestion
        const result = mock('job').get()
```
https://www.notion.so/pkhing/Testing-e6908a78c4f94c2f9c8e3510130332d4?pvs=4#7393d7aef39b4186a3aed616f64259a1